hexrd/distortion/distortionabc.py

```diff
@@ -8,9 +8,7 @@ class DistortionABC(metaclass=abc.ABCMeta):
     @abc.abstractmethod
     def apply(self, xy_in):
         """Apply distortion mapping"""
-        pass
 
     @abc.abstractmethod
     def apply_inverse(self, xy_in):
         """Apply inverse distortion mapping"""
-        pass
```

Comment:
This pass is necessary, please either do not remove it, or raise a `NotImplementedError`.

Reply:
See previous comment